pkg/commands/workload_apply_test.go

```diff
@@ -7124,6 +7136,466 @@ To get status: "tanzu apps workload get spring-petclinic"
 			},
 			ShouldError: true,
 		},
+		{
+			Name:                "create from local source using lsp",
+			Skip:                runtm.GOOS == "windows",
```

Comment:
the _Publishing_ output message changes based on the OS, for Windows it uses `\` but it can not be easily replaced because working with strings it becomes `\\`.